docs/developers/tutorials/your-first-dapp.md

```diff
@@ -253,41 +197,35 @@ cd dapp
 
 ### **Configure the app**
 
-Customize the Smart Contract Address
+Use the prefered editor and customize the Smart Contract Address located in `src/config/config-devnet.tsx`
 
 ```sh
-nano src/config.tsx
+code .
 ```
 
-We'll take a look on the first instruction:
+Then edit this instruction, and change it to the contract address that was shown after mxpy contract deploy:
 
-```javascript
-export const contractAddress = "erd1qqqqqqqqqqqqq...";
-```
+![img](/developers/tutorial/config-screenshot.png)
 
-and we'll change it to our contract address that was shown after mxpy contract deploy:
-
-```javascript
-export const contractAddress =
-  "erd1qqqqqqqqqqqqqpgq0hmfvuygs34cgqsvg...ffh4y04cysagr6cn";
-```
-
-Save and close `config.tsx` and we're ready for the first build.
 
 [comment]: # (mx-context-auto)
 
 ### **Build the dApp**
 
+:::important
+[Please make sure you have **yarn installed**](https://classic.yarnpkg.com/lang/en/docs/install/#mac-stable) on your machine.
```

Comment:
The link contains the segment for mac. Perhaps remove the segment?

Reply:
fixed